src/controller/UserController.php

```diff
@@ -20,7 +20,7 @@ public function connexion(UserModel $userModel)
     {
         // The form is not submitted, posting the connexion form
         if (count($_POST) === 0) {
-            echo $this->twig->render('connexion.twig', ['SESSION' => $_SESSION]);
+            $this->render('connexion.twig', [], [], $_SESSION);
```

Comment:
Si $_SESSION doit toujours être renseigné, le mettre obligatoire dans ta fonction render.

Reply:
Modif ok : 
$this->render('connexion.twig', $_SESSION);